### ISSUE TITLE
[darknet-alexeyab-git] patch rejected

### ISSUE THREAD

**Issue by vfbsilva** [acxz/pkgbuilds#198]
I was unable to build the last version of the package as the error on the title suggests.

```
 => Iniciando prepare()... patching file src/detector.c Reversed (or previously applied) patch detected!  Skipping patch. 1 out of 1 hunk ignored -- saving rejects to file src/detector.c.rej ==> ERROR: A failured happened  in prepare().
```

**Comment by acxz** [comment]
Please use proper formatting and etiquette @vfbsilva when posting.

**Comment by acxz** [comment]
resolved with ad38acd
